packages/shared/src/components/profile/ReadingStreaksWidget.tsx

```diff
@@ -0,0 +1,50 @@
+import React, { ReactElement } from 'react';
+import { ActivityContainer, ActivitySectionHeader } from './ActivitySection';
+import { ReadingStreakIcon } from '../icons';
+import { UserStreak } from '../../graphql/users';
+
+interface StreakTagProps {
+  streak: number;
+  title: string;
+  isLoading: boolean;
+}
+
+interface ReadingStreaksWidgetProps {
+  streak: UserStreak;
+  isLoading: boolean;
+}
+const StreakTag = ({ streak, title, isLoading }: StreakTagProps) => {
+  return (
+    <div className="relative flex w-40 flex-col rounded-10 border border-background-subtle p-3">
```

Comment:
The w-40 is intentional? what about large streaks?

Reply:
It would need a really big number for it not to fit. But we can use the large nr format if we want

Reply:
![image](https://github.com/dailydotdev/apps/assets/1681525/71e19958-d813-4a07-b1fe-9038a24a0968)

It's an issue for future team members of daily.dev to figure out. Hopefully in 27 million years the codebase has been refactored a few times by then.

Reply:
Applied the formatting